src/v2/cookbook/creating-custom-scroll-directives.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Creating Custom Scroll Directives
+type: cookbook
+order: 5
+---
+
+## Simple Example
+
+There are many times that we might want to add a bit of behavior, especially animation, to a scroll event on a site. There are many ways to do so, but the path with the least amount of code and dependencies is to use a [custom directive](https://vuejs.org/v2/guide/custom-directive.html) to create a hook for anything that fires off a particular scroll event.
+
+``` js
+Vue.directive('scroll', {
+  inserted: function(el, binding) {
+    let f = function(evt) {
+      if (binding.value(evt, el)) {
+        window.removeEventListener('scroll', f);
+      }
+    };
+    window.addEventListener('scroll', f);
+  },
+});
+
+// main app
+new Vue({
+  el: '#app',
+  methods: {
+   handleScroll: function(evt, el) {
+    if (window.scrollY > 50) {
+      el.setAttribute("style", "opacity: 1; transform: translate3d(0, -10px, 0)")
+    }
+    return window.scrollY > 100;
+    }
+  }
+});
+```
+
+``` html
+<div id="app">
+  <h1 class="centered">Scroll me</h1>
+  <div class="box" v-scroll="handleScroll">
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A atque amet harum aut ab veritatis earum porro praesentium ut corporis. Quasi provident dolorem officia iure fugiat, eius mollitia sequi quisquam.</p>
+  </div>
+</div>
+```
+
+<p class="tip">Remember! The directive must be registered before the Vue instance</p>
```

Comment:
Maybe finish the second sentence with a period?

Reply:
👍